docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-hard-vs-soft-constraints-con.adoc

```diff
@@ -0,0 +1,26 @@
+[id='optimizer-hard-vs-soft-constraints-con']
+= A planning problem has (hard and soft) constraints
```

Comment:
Why (hard and soft)? Why not ...has hard and soft contraints?

Reply:
I guess because sometimes they can have only hard constraints, or sometimes hard, medium, and soft constraints. It's probably better as just constraints, so I'll change the title to that.